content/blog/how-elevate-supports-business-needs.md

```diff
@@ -0,0 +1,182 @@
+
+## whoami and a bit about our systems
+
+I'm David from the Netherlands and I try to spread the feeling I experience myself while working within the hosting industry whenever I can, it really is something magical!
+My employer Bizway, a Dutch Managed Service Provider (MSP from now on) gives me the chance to explore all the different aspects that are on the table when providing services to customers.
+
+Things that interest me a lot are automation, mostly anything opensource, networking and infrastructure.
+
+The first Linux I ever used was actually Debian, which I luckily still use a lot too, as it is the base for [Proxmox Virtual Environment](https://www.proxmox.com).
+While for the past 4 years CentOS and AlmaLinux took most of my time, Debian always kept a special place because of one simple reason, Debian provides in-place OS upgrades.
+
+Even though servers can, and one can discuss, maybe even should be configured in state so you can destroy and replace it easily, that is not always the case.
+
+So when AlmaLinux came around and further down the road ELevate popped up offering in-place upgrades from CentOS 7 systems to for example AlmaLinux 8 and higher, it also entered that special place!
+
+## Why there can be (and are!) still CentOS 6 servers alive and kicking
+We always used CentOS as our main operating system for our webhosting platforms, while we were eventually forced to look elsewhere, we still had a lot of CentOS 7 and a couple of CentOS 6 systems around,
+since ELevate came around we started upgrading our whole fleet to AlmaLinux 8 as we started using that for our new deployments too, we're almost done doing so.
+
+Upgrading is most of the time the less painfull way to get your customer to use a new OS,
+as with migrating to a new server there's always something that takes some debugging where in-place upgrades keep it mostly intact,
+which is important, because as a techie we tend to forget that customers mostly want the servers to just work to support their business needs.
+
+That still doesn't make up for the CentOS 6 servers currently around, which in all honesty should've been gone a long time ago already,
+in which as a MSP you can only do so much for your customer,
+if in the end they use an application for their business that for some reason only runs on CentOS 6,
+or their developer left a long time ago, well, good luck with replacing that server...
+
+It gives however a great reason to have some fun with the new release of [ELevate](https://almalinux.org/blog/2024-04-25-elevate-supports-centos-6-to-centos-7/), as it now supports ELevating CentOS 6 too!
+
+Nowadays there are less and less excuses to make, as [DirectAdmin](https://www.directadmin.com) for example made it even possible to run php 5.6 code on AlmaLinux 9,
+which is a great way of offering your customer a up-to-date operating system if they're really dependent on php 5.6, which gives us one less thing to worry about.
```

Comment:
```suggestion
Nowadays there are fewer and fewer excuses to prevent upgrades, as [DirectAdmin](https://www.directadmin.com) for example made it even possible to run PHP 5.6 code on AlmaLinux 9. It's a great way of offering your customers an up-to-date operating system if they're really dependent on PHP 5.6. It gives us one less thing to worry about.
```